test/library/packages/TOML/test/newline.chpl

```diff
@@ -0,0 +1,15 @@
+
+use TOML;
+
+config const str: string = "[owner] \
```

Comment:
Utilizing multi-line strings might make this example a little easier to follow. Also, I think it's a good idea to include an extra field to confirm the line reported isn't just the last line number, e.g.


```chpl
config const str = """[owner]

                   name = "Foo Bar"
                   dob = 2000-04-30-234-42
                   field = "someValue"
                   """;
```

Reply:
I've made the changes.